src/VisualStudio/Core/Impl/ProjectSystem/CPS/TempPECompiler.cs

```diff
@@ -65,19 +73,15 @@ public async Task<bool> CompileAsync(IWorkspaceProjectContext context, string ou
                     // Turn off any warnings as errors just in case
                     .WithGeneralDiagnosticOption(ReportDiagnostic.Suppress)
                     .WithReportSuppressedDiagnostics(false)
-                    .WithSpecificDiagnosticOptions(null)
+                    .WithSpecificDiagnosticOptions(ImmutableDictionary<string, ReportDiagnostic>.Empty)
```

Comment:
FYI to @dotnet/roslyn-compiler: WithSpecificDiagnosticOptions is now marked as non-null accepting. Should that be reannotated as well?

Reply:
Yes it should. That explicitly allows `null`

```
            if (values == null)
            {
                values = ImmutableDictionary<string, ReportDiagnostic>.Empty;
            }
```

Reply:
Changed.